GLOCKFILE

```diff
@@ -1,19 +0,0 @@
-cmd github.com/golang/lint/golint
-cmd github.com/jteeuwen/go-bindata/go-bindata
-cmd github.com/mitchellh/gox
```

Comment:
[gox](https://github.com/mitchellh/gox) is used by `make docker-cross` but this dependency did not get migrated to the module files. We need `make docker-cross` to succeed to do releases. I'm thinking the easiest way to deal with this would be to run `go install github.com/mitchellh/gox` and commit the updated module files.

Reply:
I've added it as part of of the Dockerfile so that it exist there. I want to avoid bloating the modules with un-needed dependencies (as anything that would import this project would then import the extra module). For the longer term though, I'd recommend switching to [GoReleaser](https://goreleaser.com/). WDYT?

Reply:
Works for me. Let's chat separately about release options!